enhancements/microshift/microshift-router-namespaces.md

```diff
@@ -0,0 +1,119 @@
+---
+title: microshift-route-admission-policy
+authors:
+  - "@pacevedom"
+reviewers:
+  - "@eslutsky"
+  - "@copejon"
+  - "@ggiguash"
+  - "@pmtk"
+  - "@pliurh"
+  - "@jerpeter1"
+  - "@Miciah"
+approvers:
+  - "@dhellmann"
+creation-date: 2024-01-30
+last-updated: 2024-01-31
+tracking-link:
+  - https://issues.redhat.com/browse/OCPSTRAT-1067
+see-also:
+  - "/enhancements/ingress/openshift-route-admission-policy.md"
+---
+
+# MicroShift route admission policy
+## Summary
+OpenShift defaults to not allow routes in multiple namespaces use the same
+hostname, and MicroShift inherits that default.
+
+Ever since OpenShift 4 this has been possible to configure, and MicroShift
+should allow that too to accommodate additional use cases.
+
+## Motivation
+Users who compose applications from multiple sources may deploy those to
+different namespaces, albeit want to expose them using the same hostname and
+different path.
+
+### User Stories
+As a MicroShift admin, I want to configure MicroShift to support combining
+routes from multiple namespaces.
+
+### Goals
+- Allow the use of the same host in routes from different namespaces.
```

Comment:
Did we decide whether or not to change the default for MicroShift?

Reply:
Yes, its the opposite for MicroShift than OCP. This is also included.

Reply:
I didn't find where that was mentioned in the document.

Reply:
Done